src/styles/highlighter.less

```diff
@@ -51,4 +51,46 @@
     &unresolved &game {
         background-color: #7D9CAD;
     }
+
+    &tooltip {
+        position: relative;
+    }
+
+    &tooltiptext {
+        visibility: hidden;
+        width: 120px;
+        text-align: center;
+        padding: 5px 0;
+        border-radius: 6px;
+        border: 2px solid #000000;
+    
+        /* Position the tooltip text - see examples below! */
+        position: absolute;
+        z-index: 1;
+
+        &-won {
+            background-color: #799B29;
+            color: #000000;
+        }
+
+        &-lost {
+            background-color: #D97962;
+            color: #000000;
+        }
+
+        &-jigo {
+            background-color: #D7C3D1;
+            color: #000000;
+        }
+
+        &-unresolved {
+            background-color: #CBDCE5;
+            color: #000000;
+        }
+    }
+
+    /* Show the tooltip text when you mouse over the tooltip container */
```

Comment:
Funny thing - the comment make _less_ generate unwanted code - [see here](https://github.com/barcicki/go-results-highlighter/pull/3/files#diff-e531488a8df12ea1c78a28f80b8ed727R1).